README.md

```diff
@@ -41,8 +41,8 @@ Alternatively, you can supply your own copy of the root CA.
 
 ## License
 
-Copyright &copy; 2021 Stojan Dimitrovski. Licensed under the MIT License. See
-`LICENSE` for more information.
+Copyright &copy; 2021 Stojan Dimitrovski. 
```

Comment:
I think we could add to this line ` 2024 BLOCKY Inc.`. If that's not a clear yes, then maybe an engineering item?

Reply:
Done